openreview/venue_request/process/matching_pre_process.py

```diff
@@ -1,4 +1,21 @@
 def process(client, note, invitation):
+    _, baseurl_v2 = openreview.tools.get_base_urls(client)
+    client_v2 = openreview.api.OpenReviewClient(baseurl=baseurl_v2, token=client.token)
+    forum_note = client.get_note(note.forum)
+    venue_id = forum_note.content.get('venue_id', '')
+    domain = client_v2.get_group(venue_id)
+    submission_name = domain.content.get('submission_name', '').get('value')
+    senior_area_chairs_name = domain.content.get('senior_area_chairs_name', {}).get('value')
 
-    if 'Yes' in note.content['compute_affinity_scores'] and 'upload_affinity_scores' in note.content:
-        raise openreview.OpenReviewException('Either upload your own affinity scores or select affinity scores computed by OpenReview')
\ No newline at end of file
+    submissions_over_2000 = client_v2.get_notes(invitation=f'{venue_id}/-/{submission_name}', offset=2000)
```

Comment:
Check get_all_notes and the parameter `with_count`, it can help you to get the count without using the offset.

Reply:
I would use the venueid here to get the active submissions, using the invitation means we will get withdrawn/desk-rejected as well

Reply:
@melisabok for this call, I'm worried about venues with a large number of submissions and that it could take several seconds before PCs receive any feedback. It seems like `offset` with `get_notes()` is significantly faster and gets a response in < 1s. What do you think?

Reply:
ok! I thought there was a call to get 1 only paper and get the count. 

For example: api2.openreview.net/notes?invitation=ICML.cc/2023/Conference/-/Submission&limit=1

you can see the count in the response. It seems the python function doesn't read that data. 

you can implement a new function called `get_notes_count` and read the count from the json response.

Reply:
Ahh okay I see what you mean now. Then I think we can do:
`client_v2.get_notes(content={ 'venueid':submission_venue_id }, limit=1, with_count=True)`
which returns 1 submission note & the total count. I didn't realize we could do it this way.

If we can get count like this, do you still think a `get_notes_count()` function is needed?

Reply:
you can use the first option, no need to implement a new function.